fares-data-build-tool/create-data
# fix: Netex name bug and updating product dates bug

## Description

Fixed issue where netex files were being created with '/' in the name (if the user has a / in their product name), and the bug where product dates werent being updated in the DB table when a user updates them through the edit feature.

Also, removing all feature flags for multi operator.

## Testing instructions

Ensure editing products' dates works correctly, and export still works fine (on test env).
